chainstate/storage/src/schema.rs

```diff
@@ -69,6 +69,8 @@ storage::decl_schema! {
         pub DBAccountingPoolDataTip: Map<PoolId, PoolData>,
         /// Store for tip accounting pool balances
         pub DBAccountingPoolBalancesTip: Map<PoolId, Amount>,
+        /// Store for tip accounting pool owner balances
+        pub DBAccountingPoolOwnerBalancesTip: Map<PoolId, Amount>,
```

Comment:
So at the lowest level, this probably goes here... why not include this value in PoolData?

Reply:
hmm, right, didn't see that option. Will do